### PR TITLE
[Osquery] OSQuery DFIR Saved Queries

### PR DESCRIPTION
**Pull request by w0rk3r**
## What does this PR do?

Adds DFIR-related saved queries to the OSQuery manager integration

### CODE REVIEW

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-157d5550-fd27-11ec-8645-83a23bc513b5.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "attributes": {
+        "created_at": "2022-07-06T12:28:04.772Z",
+        "created_by": "elastic",
+        "description": "Get the exclusions configured in Microsoft Windows Defender and their details.",
+        "id": "defender_exclusions_elastic",
+        "interval": "3600",
+        "platform": "windows",
+        "query": "select name, regex_match(r.key, '[^\\\\]\\w+$', 0) as ExclusionType, key, datetime(mtime, 'unixepoch', 'UTC') as ModificationTimeUTC from registry r where\nr.key == 'HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\Windows Defender\\\\Exclusions\\\\Extensions' or\nr.key == 'HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\Windows Defender\\\\Exclusions\\\\IpAddresses' or\nr.key == 'HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\Windows Defender\\\\Exclusions\\\\Paths' or\nr.key == 'HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\Windows Defender\\\\Exclusions\\\\Processes' or\nr.key == 'HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\Windows Defender\\\\Exclusions\\\\TemporaryPaths'",
```
Comment (tomsonpl) [inline packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-157d5550-fd27-11ec-8645-83a23bc513b5.json:9]: I tested the query and it gets installed with integration  :) 👍  I would just suggest to put SELECT , FROM and OR in CAPITAL case.

**Review by tomsonpl — CHANGES_REQUESTED** [review]
Left one comments, besides LGTM from Asset management :)

**Review by melissaburpo — COMMENTED** [review]
I just had one small suggested change to update the `id`. 

Also, a question: should we add ECS mappings for any fields returned in the results? I tried testing the query to see what the results look like, but I wasn't able to run it successfully - I suspect that's because in its current form the query has some extra formatting for the json version of the saved object.

**Comment by w0rk3r** [comment]
@melissaburpo did it returned any error? Because it will return null if no exceptions exist

**Comment by melissaburpo** [comment]
The issue was that I tried to copy/paste the query directly from the PR to try running it, so it had some syntax issues (like the `\nr` characters). 

Thanks for sharing the screenshot though, that helps! Looking at the results, I don't think there are any ECS mappings needed for the `defender_exclusions` query, so that helps to answer that question. Appreciate it!

**Comment by w0rk3r** [comment]
Here are a few screenshots on the queries returning the desired results:

<details>
<summary>defender_exclusions_windows_elastic</summary>

image

</details>

<details>
<summary>posh_logging_windows_elastic</summary>

image

</details>

<details>
<summary>unsigned_services_vt_windows_elastic</summary>

image

</details>

<details>
<summary>unsigned_processes_vt_windows_elastic</summary>

image

</details>

<details>
<summary>unsigned_startup_items_vt_windows_elastic</summary>

image

</details>

<details>
<summary>unsigned_dlls_on_system_folders_vt_windows_elastic</summary>

image

</details>

<details>
<summary>winbaseobj_mutex_search_windows_elastic</summary>

image

</details>

<details>
<summary>executables_or_drivers_in_temp_folder_vt_windows_elastic</summary>

image

</details>

<details>
<summary>wdigest_uselogoncredential_windows_elastic</summary>

image

</details>

Let me know if I can help with anything around these ;)

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-0f61edf0-17e1-11ed-89c6-331eb0db6d01.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "interval": "3600",
+        "platform": "windows",
+        "query": "SELECT object_name, object_type, session_id FROM winbaseobj WHERE object_type == \"Mutant\" AND object_name LIKE '%InsertMutex_Here%'",
+        "updated_at": "2022-08-09T12:44:51.150Z",
+        "updated_by": "elastic"
+    },
+    "coreMigrationVersion": "8.3.3",
+    "id": "osquery_manager-0f61edf0-17e1-11ed-89c6-331eb0db6d01",
+    "references": [
+        "https://www.sans.org/blog/looking-at-mutex-objects-for-malware-discovery-indicators-of-compromise/"
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-0f61edf0-17e1-11ed-89c6-331eb0db6d01.json:16]: Not obligatory, just saw it available and thought the use could benefit of some additional context. I can remove them

**Review by tomsonpl — APPROVED** [review]
LGTM

**Comment by tomsonpl** [comment]
@w0rk3r I think you can merge it. Let's then test the integration on staging before going on production :)

**Comment by tomsonpl** [comment]
https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/main/1201/pipeline/ this is a job triggered from this PR. Let's see what happens next 🤞

**Pull request merged**